src/frontend/src/flows/dappsExplorer/dapps.ts

```diff
@@ -43,7 +43,7 @@ export class KnownDapp {
     return this.descr.oneLiner;
   }
 
-  public get website(): string | undefined {
+  public get website(): string {
```

Comment:
lit-analyzer noticed that an `href` was being set to a potentially undefined string which came from here. Turns out the website cannot be undefined.